test/input_gen/transLayer.py

```diff
@@ -144,6 +144,36 @@ def to_nntr_weights(self, weights):
     K.layers.MaxPool2D,
 )
 
+##
+# @brief Multiout wrapper layer, this class separate gradient
+# to calculate derivative properly
+# when calling, this returns [x] * @a num_output just like output Layer does in NNTrainer
+class MultiOutLayer(IdentityTransLayer):
+
+    ##
+    # @brief init function
+    # @param tf_layer tf_layer to get number of outputs
+    # @param num_output explicit number to generate number of output
+    def __init__(self, tf_layer = None, *args, num_output, **kwargs):
+        if not tf_layer:
+            tf_layer = K.layers.Layer()
+
+        super().__init__(tf_layer, *args, **kwargs)
+
+        # this enables seperating gradient one by one
+        self.stub_layers = [K.layers.Lambda(lambda x: x + 0) for i in range(num_output)]
```

Comment:
How about simply
```suggestion
        self.stub_layers = [K.layers.Lambda(lambda x: x) for i in range(num_output)]
```

Reply:
Checked this, and it didn't work. I suppose if I don't do x: x + 0, I think tensorflow optimizer optimizes this to a `noop`.

Reply:
ok.